crates/solidity/testing/perf/npm/src/benchmarks/common.mts

```diff
@@ -0,0 +1,137 @@
+import fs from "node:fs";
+import path from "node:path";
+
+export const hasGC = typeof global.gc == "function";
+
+export async function runGC() {
+  if (hasGC) {
+    global.gc!();
+    await sleep(100);
+  }
+}
+
+export function sleep(ms: number): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+export function checkCI() {
+  if (process.env["CI"] == undefined) {
+    console.error("Must run with CI=true");
+  }
+}
+
+export function round2(n: number): number {
+  return Math.round(n * 100) / 100;
+}
+
+export const verboseOption = "verbose";
+
+const verbose = process.argv.includes(verboseOption);
```

Comment:
wouldn't that check for `verbose` not `--verbose` in `argv`? how is that used?
I suggest removing it if was no longer used/needed.